### ISSUE TITLE
New Content Management System (CMS) features have accessibility issues

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#5340]
## Describe the bug
This bug was discovered during testing of #5106.

The new forms added for Content Management System (CMS) features have accessibility issues.  Namely "Form elements must have labels" is flagged by the Chrome "axe DevTools" plugin, along with a few other minor issues.

## To Reproduce
1. Open up one of the new forms added for Content Management System (CMS) features.  For example, open the "Edit User Agreement" form.
2. Run an accessibility scanner on the page, e.g. Chrome's "axe DevTools" plugin
3. You'll see a number of errors:
    * Each text area is missing an official label.  So you'll see a large number of "Form elements must have labels" errors.
    * The blue note at the top of the page also has minor accessibility issues.  The "Markdown" link will throw two errors:
        * Elements must meet minimum color contrast ratio threshold (Fix by increasing the color contrast, e.g. by making the link text darker or similar)
        * Links must be distinguishable without relying on color.  (Fix by making the link underlined or similar)

## Expected behavior
These forms should pass a basic accessibility scanner like Chrome's "axe DevTools" plugin

## Related work
Introduced in #5106
